src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+table.tHead.addEventListener('click', (ev) => {
```

Comment:
```suggestion
table.tHead.addEventListener('click', (event) => {
```
It is better to use full, informative names. especially if it's about 2-3 extra chars